vue/src/components/cms/PermissionPanel.vue

```diff
@@ -1,51 +1,69 @@
+<!-- This panel shows a list of user rights. They can be granted and deprived separately -->
```

Comment:
"This panel shows a list of user rights. They can be given and revoked separately"